Formula/osm2pgsql.rb

```diff
@@ -31,13 +31,8 @@ def install
     inreplace "cmake/FindLua.cmake", /set\(LUA_VERSIONS5( \d\.\d)+\)/,
                                      "set(LUA_VERSIONS5 #{lua_version})"
 
-    # Use Proj 6.0.0 compatibility headers
-    # https://github.com/openstreetmap/osm2pgsql/issues/922
-    # and https://github.com/osmcode/libosmium/issues/277
-    ENV.append_to_cflags "-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H"
-
     mkdir "build" do
-      system "cmake", "-DWITH_LUAJIT=ON", "..", *std_cmake_args
+      system "cmake", "-DWITH_LUAJIT=ON", "-DUSE_PROJ_LIB=6", "..", *std_cmake_args
```

Comment:
Where does the `6` come from?

Reply:
As described here: https://github.com/openstreetmap/osm2pgsql/commit/552e4bc0b874989878ea529193d106eeaed2e7e5, the old API for PROJ was actually implemented in PROJ 4, and the new API debuted in PROJ 6.  Both APIs were supported in PROJ 6 and 7, but in PROJ 8, the old PROJ API was removed.  So the number here is referring the version in which the API was first implemented, not the version currently being used.